tools/src/main/java/org/apache/rocketmq/tools/command/cluster/ClusterListSubCommand.java

```diff
@@ -113,7 +113,7 @@ private void printClusterMoreStats(final DefaultMQAdminExt defaultMQAdminExt) th
         while (itCluster.hasNext()) {
             Map.Entry<String, Set<String>> next = itCluster.next();
             String clusterName = next.getKey();
-            TreeSet<String> brokerNameSet = new TreeSet<String>();
```

Comment:
I think tools module should be consistent with broker module. There is no need to maintain low version compatibility.